ui/app/components/sidebar/user-menu.hbs

```diff
@@ -7,7 +7,7 @@
   as |Dropdown|
 >
   <Dropdown.Trigger class="has-pointer has-text-white" data-test-user-menu-trigger>
```

Comment:
@zofskeez I _think_ the classes `has-pointer has-text-white` here are not needed anymore

Reply:
Good catch I will remove those!